aiidalab_widgets_base/computational_resources.py

```diff
@@ -888,11 +906,13 @@ def _reset(self):
         self.work_dir.value = ""
         self.mpirun_command.value = "mpirun -n {tot_num_mpiprocs}"
         self.mpiprocs_per_machine.value = 1
-        self.transport.value = "ssh"
+        self.default_memory_per_machine.value = 16000000
```

Comment:
Where is this default value coming from?

Reply:
It's should be set for AiiDA 2.x, I think @yakutovicha add it to the computer setup and made it mandatory.
It is hard to say if we should provide the default value for this field, maybe we shouldn't provide the default one. 
But this comes with the problem that the code-registry-database doesn't have this field set yet, the "quick setup" of widget will fail since no value is provided.

Reply:
I don't really think it matters where it comes from, but I think there should be some kind of justification for it and then it should be documented here in the form of a short comment.